Preferences.sublime-settings

```diff
@@ -0,0 +1,3 @@
+{
+  "ctags.opts": []
+}
```

Comment:
Did you intend to add this file? Is it necessary?

Reply:
I intended to add it, but just noticed it's not necessary, it works without it.